src/OrbitGl/App.cpp

```diff
@@ -2481,10 +2482,47 @@ Future<std::vector<ErrorMessageOr<CanceledOr<void>>>> OrbitApp::LoadAllSymbols()
 
     loading_futures.push_back(RetrieveModuleAndLoadSymbols(module));
   }
+  if (absl::GetFlag(FLAGS_auto_frame_track)) {
+    // Orbit will try to add the default frame track while loading all symbols.
+    std::ignore = AddDefaultFrameTrack();
```

Comment:
Nit: I don't like the use of std::ignore. Returning an error message from this method suggest that the error has not yet been handled - maybe move the writing of the error log out of "AddDefaultFrameTrack"?

Reply:
This here depends on one important distinction: Do we want to display a message if something goes wrong, or not. If we want to, then I would show it here. But I think it does not make sense to have a popup, so I would change the code to the following:

`Future<void> AddDefaultFrameTrackOrLogError();` or just
`void AddDefaultFrameTrackOrLogError();`

And then log the error in every case, which is currently not happening. With the current code, in the case LoadPreset returns an ErrorMessage this is not logged (Because of this, I request changes, otherwise LGTM)